test/src/integration/integration_scalardl_test.go

```diff
@@ -44,6 +46,10 @@ func TestScalarDLWithJavaClientExpectStatusCodeIsValid(t *testing.T) {
 	logger.Logf(t, "URL: %s", scalarurl)
 	writePropertiesFile(t, scalarurl)
 
+	if ! isReachable(t, scalarurl + ":50051") {
```

Comment:
I think you havne't done `gofmt` for this. Please do!

Reply:
fixed in 0e0495f.